src/utils/XhrNetworkInterceptor.ts

```diff
@@ -98,26 +99,33 @@ export const injectHeaders = async (
 
   const headerFound = networkData.requestHeaders.traceparent != null;
 
-  networkData.w3cc = headerFound;
-  headerFound
+  networkData.isW3cHeaderFound = headerFound;
+  const injectionMethodology = headerFound
     ? IdentifyCaughtHeader(networkData, w3c_caught_header)
     : injectGeneratedData(networkData, w3c_generated_header);
+  return injectionMethodology;
 };
 
-const IdentifyCaughtHeader = (networkData: NetworkData, w3c_caught_header: boolean) => {
-  w3c_caught_header && (networkData.wceti = networkData.requestHeaders.traceparent);
+const IdentifyCaughtHeader = async (networkData: NetworkData, w3c_caught_header: boolean) => {
```

Comment:
Let's follow `camelCase` here as well.
```suggestion
const identifyCaughtHeader = async (networkData: NetworkData, w3cCaughtHeader: boolean) => {
```